cava/arrow-examples/ArrowExamples.hs

```diff
@@ -0,0 +1,243 @@
+{-# OPTIONS_GHC -cpp -XMagicHash #-}
```

Comment:
This is the offending generated file

Reply:
Sorry, what's going wrong here? Thank you.

Reply:
This file is autogenerated, it's now covered by your modification to the .gitignore, but is still in the PR (file should be removed I think)